src/portable/renesas/rusb2/rusb2_type.h

```diff
@@ -40,7 +40,9 @@ extern "C" {
 /* Start of definition of packed structs (used by the CCRX toolchain) */
 TU_ATTR_PACKED_BEGIN
 TU_ATTR_BIT_FIELD_ORDER_BEGIN
-
+#if defined(__ICCARM__)
+  __packed
```

Comment:
Yes, I am using the -e flag, for IAR workbench users it is under :
Options/ C_C++ compiler/ Language 1 / Language conformance / Standard with IAR extensions

I have the same question regarding TU_ATTR_PACKED, I thought it should be enough. However, I am using IAR compiler 9.1, which seems insufficient for these structs. When I use the __packed I am getting rid of the warnings. Here is IAR warning that I am getting:
![iar_warn](https://github.com/hathach/tinyusb/assets/36272850/bb654681-ddd3-4481-a9c5-d60a441a245c)

Reply:
Here is an example given by IAR, which somehow makes it seems that only __packed is supported.
https://www.iar.com/contentassets/e20cfa542d1a4734aa6b234bea42f11a/example-5.pdf
Another link that led me to use __packed on these structs:
https://www.iar.com/knowledge/support/technical-notes/compiler/accessing-unaligned-data/

Reply:
that is strange, I don't often use IAR, but this changes need further comfirmation. I would rather change the code a bit than introducing #ifdef that breaks the TU_ATTR_. If you have no hurry, please wait until I get time to set up IAR to test with. Otherwise, please revert this and make it into its own PR so that we can merge the ctz first.

Reply:
I split the commits, so you can pick the other commit, and come back to the other one when you have more info

Thank you a lot

Reply:
Thanks but what I really mean is to split them into 2 seperated PRs. Spliting commits don't really work though.

Reply:
done